benchmark/bench-report/bin/bench-runner.sh

```diff
@@ -176,15 +178,28 @@ bench_exec_one() {
   local BENCH_NAME1
   local BENCH_NAME2
   local BENCH_NAME
+  # XXX this is a hack to make the "/" separated names used in the functions
+  # determining options based on benchmark name. For tasty-bench the benchmark
+  # names are separated by "." instead of "/".
```

Comment:
Can you give a small example of what this hack script intends to do?

Reply:
I will try adding an example.